joblib/memory.py

```diff
@@ -849,10 +849,7 @@ def _persist_input(self, duration, args, kwargs, this_duration_limit=0.5):
                           "(results will be correct in all cases). \n"
                           "The reason for this is probably some large input "
                           "arguments for a wrapped\n"
-                          " function (e.g. large strings).\n"
-                          "THIS IS A JOBLIB ISSUE. If you can, kindly provide "
-                          "the joblib's team with an\n"
-                          " example so that they can fix the problem."
+                          " function."
```

Comment:
The locations of `\n` signs make no sense to me. Let's remove them all from this message and let the terminal of the user wrap the warning message instead.